app/controllers/hotels_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HotelsController < ApplicationController
+	def show
+		@hotel = Hotel.find(params[:id])
+		@unique = @hotel.unique_guests
```

Comment:
NIce job making a method here. You can also call this method directly from the View, you don't necessarily have to make another instance variable for it.